src/PHP8Fun.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @param $a
+ * @param $b
+ * Call str_contains!
+ * @return bool
+ */
+function find_my_string( $a, $b ): bool {
+	if ( $x = str_contains( $a, $b ) ) {
+		echo 'The string has been found';
+	} else {
+		echo "String not found";
+	}
+
+	return $x;
+}
+$result2 = find_my_string('a', 'b');
+var_dump( $result2 );
```

Comment:
:warning: **Warning**: var_dump() found. Debug code should not normally be used in production (*WordPress.PHP.DevelopmentFunctions.error_log_var_dump*).